<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab1Window</class>
 <widget class="QWidget" name="Lab1Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #212121;
    color: #eeeeee;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}
QGroupBox {
    border: 1px solid #757575;
    border-radius: 5px;
    margin-top: 10px;
    font-weight: bold;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    color: #ff9800; /* Помаранчевий акцент */
}
QPushButton {
    background-color: #ff5722; /* Яскраво-оранжевий */
    color: white;
    font-weight: bold;
    border-radius: 5px;
    padding: 10px;
}
QPushButton:hover { background-color: #e64a19; }
QDoubleSpinBox, QSpinBox, QTextEdit {
    background-color: #424242;
    color: white;
    border: 1px solid #616161;
    padding: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Лабораторна робота №2</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>121</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Data</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinX">
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>3.140000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinY">
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>-27</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spinZ">
      <property name="minimum">
       <double>-1000.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>-23.199999999999999</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCalc">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>260</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обчислити формулу</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLog">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>80</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
